doc/user/hierarchy.xml

```diff
@@ -534,17 +534,13 @@ x
 
       <para>
 
-      There are two ways to export a variable,
-      such as a construction environment,
-      from an &SConscript; file,
-      so that it may be used by other &SConscript; files.
-      First, you can call the &Export;
-      function with a list of variables,
-      or a string of white-space separated variable names.
-      Each call to &Export; adds one
-      or more variables to a global list
-      of variables that are available for import
-      by other &SConscript; files.
+      There are two ways to export a variable
+      from an &SConscript; file.
+      The first way is to call the &f-link-Export; function.
+      &Export; takes one or more arguments that are
+      string representations of the variable names - that is,
+      to export the variable <varname>env</varname>,
+      you supply the string <quote>env</quote> to &Export;:
```

Comment:
Sounds like you're dropping that Export can be called multiple times and each one adds to the list of exported values?

Reply:
not intending to. There's a paragraph later that covers the topic, but could be more explicit - I'll make a note to myself.

Reply:
Updated, hopefully it's explained better now (this appears quite a bit lower down, around line 619 in the new patched file)